app/views/layouts/govuk_admin_template.html.erb

```diff
@@ -9,7 +9,7 @@
 <!--[if gt IE 8]><!--><html class="no-js" lang="en"><!--<![endif]-->
   <head>
     <meta charset="utf-8">
-    <title><%= content_for?(:page_title) ? yield(:page_title) : "GOV.UK" %></title>
+    <title><%= content_for?(:page_title) ? yield(:page_title) : GovukAdminTemplate::Config.app_title %></title>
```

Comment:
I think line 62 needs something similar, which is where the `app_title` is rendered on the page.